src/index.css

```diff
@@ -50,6 +52,12 @@
     var(--primary-dark) 90deg,
     var(--primary-background) 0
   );
+
+  --tertiary-subheader: linear-gradient(
+    270deg,
+    var(--primary-bright) 3.96%,
```

Comment:
Why don't we use `4%` here?

Reply:
We don't use this class, so I removed it